### ISSUE TITLE
Unable to import numpy

### ISSUE THREAD

**Issue by ericmandm** [ninia/jep#224]
I'm using python 3.6.8 and Java 11.  Here is a snippet of my code that is failing:
`        try (Interpreter interp = new SharedInterpreter()) {
            interp.exec("import numpy");
            interp.exec("import json");
            ...
`
But I get the following exception:
`jep.JepException: <type 'exceptions.ImportError'>: No module named numpy
	at jep.Jep.exec(Native Method)
	at jep.Jep.exec(Jep.java:478)
`
* I am able run the python script (that I am trying to convert to jep) that successfully imports the numpy module.  If I start a python interpreter from the command line, I am also able to execute the "import numpy" without error.
* jep is able to import the json module.

Thank you for any suggestions about what I should do to get this working.
Best Regards,
Eric

**Comment by bsteffensmeier** [comment]
Where is numpy installed at? Where is the Python jep module installed? Typically these should both be installed next to eachother in the site-packages directory of your python installation however there are many other installation locations that can complicate things.

**Comment by ericmandm** [comment]
Thank you for your reply.  It led me to checking versions and locations.  It turns out that I have python 2.7 and 3.6 installed in my Unbuntu instance, so when I used "pip install jep", it must have installed a version for 2.6 and I only have numpy installed for python 3.6.  
I set my default python version to 3.6, then re-ran "pip install jep" (which installed v3.9.0) and now the import is working.
Thanks!
